src/storage-client/src/types/sources.rs

```diff
@@ -558,6 +559,14 @@ impl PathSummary<MzOffset> for MzOffset {
     }
 }
 
+impl Refines<()> for MzOffset {
+    fn to_inner(_: ()) -> Self {
+        MzOffset::default()
```

Comment:
is `MzOffset`'s default meaningful?

Reply:
Yes, it's offset zero

Reply:
I changed this to `MzOffset::minimum()` for good measure